lib/maths/CTimeSeriesModel.cc

```diff
@@ -170,15 +170,14 @@ const std::string IS_FORECASTABLE_OLD_TAG{"h"};
 
 // Anomaly model
 // Version >= 6.5
-const std::string ANOMALIES_6_5_TAG{"b"};
+const std::string ANOMALY_6_5_TAG{"b"};
 const std::string ANOMALY_FEATURE_MODEL_6_5_TAG{"d"};
 // Version < 6.5
 // Discarded on state upgrade because features have changed.
 // Anomaly model nested
-const std::string TAG_TAG{"a"};
-const std::string OPEN_TIME_TAG{"b"};
-const std::string SUM_PREDICTION_ERROR_TAG{"c"};
-const std::string MEAN_MINUS_LOG_PROBABILITY_TAG{"d"};
+const std::string OPEN_TIME_TAG{"a"};
+const std::string SUM_PREDICTION_ERROR_TAG{"b"};
+const std::string MEAN_ABS_PREDICTION_ERROR_TAG{"c"};
```

Comment:
I haven't looked that closely, but something seems wrong here, or at least confusing.  If these tags are being used to parse 6.4 state which is then discarded then surely the values need to be the same as before?  Or if these tags used to be used to parse a completely different section of the state in 6.4 to what they're now being used to parse then I think the comment on lines 175-177 should be made clearer.

Reply:
These tags are not used to parse 6.4 state. This object is never restored from 6.4, i.e. see [this](https://github.com/elastic/ml-cpp/blob/master/lib/maths/CTimeSeriesModel.cc#L575) line. I'll update the comment to make this clear.